### ISSUE TITLE
Move to 3.x?

### ISSUE THREAD

**Issue by dmlloyd** [smallrye/smallrye-common#363]
With the baseline of Java 17, it might make sense to move the base version to 3.x. WDYT?

**Comment by radcortez** [comment]
Sounds reasonable.

**Comment by dmlloyd** [comment]
Can I update just the `next-version` in the `project.yml` file, or will that kick off a release?

**Comment by radcortez** [comment]
Yes, but you need to commit it directly. It will kick off the release if the change comes from a PR:
https://github.com/smallrye/smallrye-common/blob/fe3b0cc3d9ced930ed0e843b4fbcf63b536ed708/.github/workflows/release-prepare.yml#L1-L7
